src/Aspire.Hosting.AWS/Lambda/LambdaExtensions.cs

```diff
@@ -97,32 +97,62 @@ public static class LambdaExtensions
         return resource;
     }
 
-    private static ExecutableResource AddOrGetLambdaServiceEmulatorResource(IDistributedApplicationBuilder builder)
-    {        
-        if (builder.Resources.FirstOrDefault(x => x.TryGetAnnotationsOfType<LambdaEmulatorAnnotation>(out _)) is not ExecutableResource serviceEmulator)
+    /// <summary>
+    /// Add the Lambda service emulator resource. The AddAWSLambdaFunction method will automatically add the Lambda service emulator if it hasn't
```

Comment:
could you add doc references to "AddAWSLambdaFunction" and "LambdaEmulatorOptions"

Reply:
Done